Cargo.toml

```diff
@@ -30,6 +30,10 @@ serde_yaml = "0.9.25"
 serde_json = "1.0.103"
 quote = "1.0.32"
 Inflector = "0.11.4"
+dashmap = "5.5.0"
```

Comment:
This sort of feels like ultimately there's an `lsp` feature and these dependencies are conditional on that feature?